ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/publish-mixed/apiserver-ingress-external/provision/cucushift-installer-rehearse-azure-ipi-publish-mixed-apiserver-ingress-external-provision-chain.yaml

```diff
@@ -0,0 +1,18 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-publish-mixed-apiserver-ingress-external-provision
+  steps:
+  - chain: ipi-conf-azure
+  - ref: ipi-conf-operator-publish-strategy
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
+  - chain: cucushift-installer-check
+  - ref: cucushift-installer-check-azure-publish-mixed
+  env:
+  - name: APISERVER_PUBLISH_STRATEGY
+    default: "External"
+  - name: INGRESS_PUBLISH_STRATEGY
```

Comment:
If both are external, is that the same as `publish: external` without `operatorPublishingStrategy` setting?

Reply:
yeah, it's true, I remember I discussed with Jinyun before on this, setting both as External is indeed allowed in this way so maybe we also need to test it.

Reply:
yes, it equals to` publish: external`, just to validate that installation have no issue in such configuration.